pkg/apis/componentconfig/validation/validation_pluginargs.go

```diff
@@ -19,15 +19,32 @@ package validation
 import (
 	"fmt"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/descheduler/pkg/api"
 	"sigs.k8s.io/descheduler/pkg/apis/componentconfig"
 )
 
 // ValidateRemovePodsViolatingNodeTaintsArgs validates RemovePodsViolatingNodeTaints arguments
 func ValidateRemovePodsViolatingNodeTaintsArgs(args *componentconfig.RemovePodsViolatingNodeTaintsArgs) error {
+	return validateCommonArgs(args.Namespaces, args.LabelSelector)
+}
+
+// ValidateRemoveFailedPodsArgs validates RemoveFailedPods arguments
+func ValidateRemoveFailedPodsArgs(args *componentconfig.RemoveFailedPodsArgs) error {
+	return validateCommonArgs(args.Namespaces, args.LabelSelector)
+}
+
+func validateCommonArgs(namespaces *api.Namespaces, labelSelector *metav1.LabelSelector) error {
```

Comment:
nit: I'm not sure about how useful this will be long term. I think separate functions to validate each common arg (eg, `ValidateNamespacesArgs` and `ValidateLabelSelectorArgs`) would be more descriptive and useful for different strategies.

Reply:
that's a good point :+1:
I will create a follow-up PR

Reply:
https://github.com/kubernetes-sigs/descheduler/pull/899